### DISCUSSION TITLE
The reported data does not correspond to the platform

### DISCUSSION THREAD

**Discussion by Alisa1100** [amzn/ads-advanced-tools-docs#11] [Q&A]
First question：
I am using /v2/sp/campaigns/report API to get sponsored product report，However, the obtained report data exists, and some active platforms do not exist？
Second question：
I am using /v2/sp/campaigns/report API to get sponsored product report，Query the product "cost" from July 25, 2022 to July 26, 2022; Report "cost" 10.59, but platform "cost" 10.40?

**Comment by elizafarley** [comment]
Thanks for your questions. Could you clarify what you mean by "active platforms" and "platform cost"? 

Also, we recommend trying out our new version 3 reports for Sponsored Products, as version 2 will be deprecated within the next year. Here's the [version 3 documentation](https://advertising.amazon.com/API/docs/en-us/reporting/v3/get-started).
